src/main/java/service/FruitWriteFile.java

```diff
@@ -0,0 +1,24 @@
+package service;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+
+public class FruitWriteFile implements WriteFile {
```

Comment:
same thing here with naming
```suggestion
public class FileWriterImpl implements FileWriter {
```